docs/server/source/http-client-server-api.rst

```diff
@@ -583,12 +583,10 @@ Blocks
 
 .. http:get:: /api/v1/blocks/{block_id}
 
-   Get the block with the ID ``block_id``. Any blocks, be they ``VALID``, ``UNDECIDED`` or ``INVALID`` will be
-   returned. To check a block's status independently, use the `Statuses endpoint <#status>`_.
-   To check the votes on a block, have a look at the `votes endpoint <#votes>`_.
+   Get the block with the ID ``block_id``.
 
    :param block_id: block ID
-   :type block_id: hex string
+   :type block_id: string
```

Comment:
The "Example request" and "Example response" also need to be updated. Here's a screenshot of what the rendered docs look like for this branch:

![screenshot from 2017-12-21 10-54-17](https://user-images.githubusercontent.com/1210951/34250517-58f2166e-e63d-11e7-855b-2c3bd78fef13.png)